.gitignore

```diff
@@ -6,6 +6,8 @@
 /public/bundles/
 /var/
 /vendor/
+/public/fonts/
+/public/sass/_fonts.scss
 ###< symfony/framework-bundle ###
```

Comment:
Symfony does some odd stuff with the configuration files. It manages *parts* of them marked by `### ... ###`. So in a future Symfony update these two lines might get dropped. They should be outside of any of the marked sections.